pkg/cmd/pipelinerun/list_test.go

```diff
@@ -144,92 +147,92 @@ func TestListPipelineRuns(t *testing.T) {
 	}{
 		{
 			name:      "Invalid namespace",
-			command:   command(t, prs, clock.Now(), ns, version, dc1),
+			command:   commandV1alpha1(t, prs, clock.Now(), ns, version, dc1),
```

Comment:
Maybe it would be better to update the name of the tests as opposed to the command?